tensorflow/core/util/tensor_bundle/byte_swap.h

```diff
@@ -108,6 +109,19 @@ Status ByteSwapArray(char *array, size_t bytes_per_elem, int array_len);
 // TODO(frreiss): Should this be a member of the Tensor class?
 Status ByteSwapTensor(Tensor *t);
 
+// Byte-swap a buffer in place.
+//
+// Args:
+//  buff: pointer to the buffer to be modified IN PLACE.
+//  size: size of bytes in this buffer.
+//  dtype: type of data in this buffer.
+//  num_of_elem: number of data in this buffer, set to -1 if unspecified
```

Comment:
Can you more clearly explain the meaning of the special case of passing -1 for `num_of_elem` please? When should we use -1, and what does the function do if it gets passed -1?

Reply:
Yes, I've updated the comments in the code. `num_of_elem` will be passed as -1 if it could not be obtained directly from tensor data. If arg `num_of_elem` is -1, function `ByteSwapBuffer()` will calculate the number of data based on arg `size` and `dtype`.